### PR TITLE
chore: cleanup todo comments throughout the codebase

### PR DESCRIPTION
**Pull request by YrrepNoj**
This PR removes various TODO comments from our code.

When relevant, the TODOs have been 'replaced' with GitHub issues.

### CODE REVIEW

File: .github/workflows/e2e.yaml [hunk 1]
```diff
@@ -44,7 +44,6 @@ jobs:
           with:
             python-version-file: 'pyproject.toml'
 
-        # TODO @JPERRY: We should be generating the proto files before this...
```
Comment (YrrepNoj) [inline .github/workflows/e2e.yaml:47]: https://github.com/defenseunicorns/leapfrogai/issues/371

File: .github/workflows/pytest.yaml [hunk 2]
```diff
@@ -31,7 +31,6 @@ jobs:
         with:
           python-version-file: 'pyproject.toml'
 
-      # TODO @JPERRY: This repeater image is floating in the void. Give it a home in this repo???
```
Comment (YrrepNoj) [inline .github/workflows/pytest.yaml:34]: https://github.com/defenseunicorns/leapfrogai/issues/370

File: Makefile [hunk 3]
```diff
@@ -30,7 +30,6 @@ gen-python: ## Generate the protobufs for the OpenAI typing within the leapfroga
 			--grpc_python_out=src/. \
 			src/leapfrogai_api/types/proto/leapfrogai_api/types/**/*.proto
 
-# TODO: Make the port a variable with an uncommon default
```
Comment (YrrepNoj) [inline Makefile:33]: https://github.com/defenseunicorns/leapfrogai/issues/369

File: packages/llama-cpp-python/Dockerfile [hunk 4]
```diff
@@ -10,7 +10,6 @@ ARG REPO_ID=TheBloke/SynthIA-7B-v2.0-GGUF
 ARG FILENAME=synthia-7b-v2.0.Q4_K_M.gguf
 ARG REVISION=3f65d882253d1f15a113dabf473a7c02a004d2b5
 
-# TODO: Standardize on a better approach for how we download models into our images across our packages
```
Comment (YrrepNoj) [inline packages/llama-cpp-python/Dockerfile:13]: https://github.com/defenseunicorns/leapfrogai/issues/368

File: packages/vllm/Dockerfile [hunk 5]
```diff
@@ -13,16 +13,13 @@ RUN groupadd -g 65532 vglusers && \
 WORKDIR /home/leapfrogai
 
 # grab necesary python dependencies
-# TODO @JPERRY: Get context as to why we are doing this for this Dockerfile but not our other ones
```
Comment (YrrepNoj) [inline packages/vllm/Dockerfile:16]: https://github.com/defenseunicorns/leapfrogai/issues/368 and https://github.com/defenseunicorns/leapfrogai/issues/367

File: packages/whisper/Dockerfile [hunk 6]
```diff
@@ -7,8 +7,6 @@ RUN python3.11 -m venv .venv
 ENV PATH="/leapfrogai/.venv/bin:$PATH"
 
 # download and covnert OpenAI's whisper base
-# TODO @JPERRY: Move this model download to a separate build stage (within its own FROM section)
```
Comment (YrrepNoj) [inline packages/whisper/Dockerfile:10]: https://github.com/defenseunicorns/leapfrogai/issues/368

File: packages/whisper/main.py [hunk 7]
```diff
@@ -15,7 +15,6 @@
 
 
 def make_transcribe_request(filename, task, language, temperature, prompt):
-    # TODO @JPERRY does this make sense to be in the function? This seems like something that should be done during initialization
```
Comment (YrrepNoj) [inline packages/whisper/main.py:18]: https://github.com/defenseunicorns/leapfrogai/issues/372

**Review by jalling97 — APPROVED** [review]

**Comment by gerred** [comment]
bring up to date, and for some reason it says convos aren't resolved

**Pull request merged**